_docs/ANDROID_SETUP.md

```diff
@@ -1,11 +1,36 @@
 # Grüvee Android Development Setup
 
+## Prerequisites
+
+### 🔥 Firebase
+
+#### Create and add `google-services.json` file to you project
+
+In Firebase console you need to create new application, get and add `google-services.json` file to the project:
+
+1. Head to https://console.firebase.google.com
+2. Go to `Project settings` under `settings icon` next to `Project Overview`
+3. Select `Android` under `General` tab, `Your apps` section
+4. On first step, fill in the fields for example:
+    - `Android package name`: com.gruvee
+    - `App nickname (optional)`: Gruvee
+5. On second step, click `Download google-services.json` button and then click `Next` button
+6. You can skip third and fourth step
+7. Add downloaded `google-services.json` to your `$GRUVEE_PROJECT/android/app` directory where `$GRUVEE_PROJECT` is the location of your root project.
+
+> Note: On **Windows 10** if it still sometimes says that it cannot find `google-services.json` file, restart the emulator and re-run `npm start` and `npm run android-start` commands.
+
 > **Please note that you need to start the emulator from Android Studio first before you run any commands.**
 >
 > For this just open up Android Studio and build the project, with an emulator set.
 >
 > To note: we have been developing on a Pixel 2 emulator and a Pixel 3 physical device, but feel free to use any device you'd like! At the end of the day it will be better for testing anyways.
```

Comment:
Hey would you mind creating some sort of "extra notes" section here? Seems to be a fair amount of notes haha

Reply:
I was considering doing something like that as in the future the list of could grow even more for sure 😅

Reply:
Ya we should totally start that now if you are open to setting up the infrastructure for it!

Reply:
Sure!